meson.build

```diff
@@ -74,14 +74,7 @@ if iconv_dep.found() or cc.has_function('iconv_open')
     conf.set('CONFIG_ICONV', 1)
 endif
 
-freetype_dep = dependency('freetype2', version: '>= 2.10.92', required: false)
-if not freetype_dep.found()
-    freetype_sp = subproject('freetype2',
-                             default_options: ['default_library=' + get_option('default_library'), # workaround for https://github.com/mesonbuild/meson/issues/8047
-                                               'zlib=builtin',
-                                               'png=enabled'])
-    freetype_dep = freetype_sp.get_variable('freetype2_dep')
-endif
+freetype_dep = dependency('freetype2', version: '>= 2.10.92', default_options: ['default_library=' + get_option('default_library')]) # workaround for https://github.com/mesonbuild/meson/issues/8047
 if freetype_dep.found()
```

Comment:
After this change, a configuration with `--wrap-mode=forcefallback` now fails for me in freetype with
```
Dependency freetype2 from subproject subprojects/freetype2 found: NO (Variable 'freetype_dep' not found)

meson.build:77:0: ERROR: Could not find dependency freetype_dep in subproject freetype2
```

I do not know whether this is an issue with your meson.build file or with freetype's. And ofc, `--wrap-mode=forcefallback` already failed for me before, just at a later stage inside the fontconfig subproject *(failing to find gperf)*, while the freetype subproject succeeded.

Reply:
> Are you sure it's not using a cached subproject?

I do not know how Meson does caching. Inside the project root, when no `build_tmp`-subdir currently exists, running
```sh
meson [setup] --wrap-mode=forcefallback build_tmp
```
fails with the above error.

Reply:
The caching for subprojects is in the `subprojects` directory. Just nuke the freetype one in there - you can't pull since I'm no longer even pointing at savannah. That _should_ fix your issue here.

Reply:
Nuking everything in `subprojects` not tracked by git  did fix the freetype configure issue and the previous gperf-configure issue disappeared as well.
Compilation of one of the subprojects *(libffi)*  fails though, but that doesn't seem like an issue with your libass-port.